test/client/app/components/mediaThumbnailTest.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { shallow } from "enzyme";
+import { expect } from "chai";
+import { spy } from "sinon";
+
+import MediaThumbnail from "../../../../assets/app/components/mediaThumbnail";
+
+describe("mediaThumbnail", () => {
```

Comment:
Can we have a `forEach` block here that runs through the whitelist of acceptable extensions? Then we can reuse one `it` block and cut down on code duplication.